extensions/control-plane/api/management-api/contract-definition-api/src/main/java/org/eclipse/edc/connector/api/management/contractdefinition/model/ContractDefinitionResponseDto.java

```diff
@@ -30,6 +30,7 @@ public class ContractDefinitionResponseDto extends BaseResponseDto {
     private String accessPolicyId;
     private String contractPolicyId;
     private List<CriterionDto> criteria = new ArrayList<>();
+    private long validity;
```

Comment:
It should be of type `DurationDto`, conversion is not really immediate but it shouldn't be that hard to implement

Reply:
Done as per last commit